src/Day02.kt

```diff
@@ -35,13 +21,7 @@ fun main() {
         return totalPaper
     }
 
-    fun getSmallestPerimeter(l: Int, w: Int, h: Int): Int {
-        val measurementSet        = intArrayOf(l, w, h)
-        val indexOfMax            = measurementSet.indexOf(maxOf(l, w, h))
-        val measurementPair       = remove(measurementSet, indexOfMax)
-
-        return measurementPair[0] + measurementPair[0] + measurementPair[1] + measurementPair[1]
-    }
+    fun getSmallestPerimeter(l: Int, w: Int, h: Int): Int = 2 * minOf(l + w, w + h, l + h)
```

Comment:
You had a good solution, but this is a bit cleaner. One thing to note is that we can use `minOf` to compare multiple values, and since we're just looking for the smallest pair of the three values it's easy just to check all three at once.